doc/releases/octopus.rst

```diff
@@ -1,3 +1,204 @@
+v15.2.5 Octopus
+===============
+
+This is the fifth release of the Ceph Octopus stable release series.
+This release brings a range of fixes
+across all components. We recommend that all Octopus users upgrade to this
+release.
+
+
+Notable Changes
+---------------
+
+* CephFS: Automatic static subtree partitioning policies may now be configured
+  using the new distributed and random ephemeral pinning extended attributes on
+  directories. See the documentation for more information:
+  https://docs.ceph.com/docs/master/cephfs/multimds/
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has repaired more than this many I/O errors in stored data a
+ ``OSD_TOO_MANY_REPAIRS`` health warning is generated.
+
+* Now when noscrub and/or nodeep-scrub flags are set globally or per pool,
+  scheduled scrubs of the type disabled will be aborted. All user initiated
+scrubs are NOT interrupted.
+
```

Comment:
Can we please add a note about https://tracker.ceph.com/issues/47297 which fixes a bug in https://tracker.ceph.com/issues/45401?

Reply:
updated with note

Reply:
thanks @theanalyst!